hivemind/dht/dht.py

```diff
@@ -55,6 +55,7 @@ def __init__(
         **kwargs,
     ):
         self._parent_pid = os.getpid()
+        self._my_pid = os.getpid()
```

Comment:
nit:

```suggestion
        self._origin_pid = os.getpid()
```

(for consistency with [MPFuture](https://github.com/learning-at-home/hivemind/blob/master/hivemind/utils/mpfuture.py#L98))